go.mod1

```diff
@@ -23,7 +23,11 @@ require (
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
 	github.com/pingcap/errors v0.11.5-0.20201126102027-b0a155152ca3
 	github.com/pingcap/failpoint v0.0.0-20200702092429-9f69995143ce
+<<<<<<< HEAD
 	github.com/pingcap/kvproto v0.0.0-20201126113434-70db5fb4b0dc
+=======
+	github.com/pingcap/kvproto v0.0.0-20210204074845-dd36cf2e1c6b
```

Comment:
Please use a release-4.0 kvproto.

Reply:
@lichunzhu ↑

Reply:
@overvenus @kennytm 
Could you please help merge this https://github.com/pingcap/kvproto/pull/727?